tests/unit/packaging/test_views.py

```diff
@@ -333,3 +337,93 @@ def test_edit_project_button_returns_project(self):
         assert views.edit_project_button(project, pretend.stub()) == {
             "project": project
         }
+
+
+class TestProjectLatestRedirects:
+    def check_release(self, db_request, project, release, endpoint):
+        db_request.route_path = pretend.call_recorder(
```

Comment:
Should I refactor this monkeypatch to happen within the context of an actual use of `monkeypatch` fixture? If so, should it be relocated to `conftest.py` (and probably renamed)?

Is it possible to decorate a class method as a fixture?

Reply:
Looks like yes, it _is_ possible to decorate an instance method as a fixture. Definitely seems prudent to `monkeypatch` this mock. Will revise accordingly.